model.py

```diff
@@ -492,8 +494,41 @@ def from_json(cls, json_dict):
         if json_dict == None:
             return None
 
+        print "model from json", json_dict
         return cls(
                 json_dict['_id'],
                 json_dict['admin_regions'],
                 json_dict['full_name'])
 
+
+class Merge(object):
+    def __init__(self, requester_user_id, base_player_obj_id, player_to_be_merged_obj_id, time, id=None):
+        self.requester_user_id = requester_user_id
+        self.base_player_obj_id = base_player_obj_id
+        self.player_to_be_merged_obj_id = player_to_be_merged_obj_id
```

Comment:
base_player vs player_to_be_merged is still a little unclear to me. can you add a comment about which player actually disappears after this merge request is completed?